### ISSUE TITLE
Display numbers in random order.

### ISSUE THREAD

**Issue by mekery** [huextrat/numeric_keyboard#2]
How about provide an option to display numbers in random order?

**Comment by huextrat** [comment]
Hi, can you describe a little more what you want. You'll want a `random` property that places the numbers 0 to 9 randomly? Can you give me a case where that would be useful I can't see a use case right now.

**Comment by mekery** [comment]
> Hi, can you describe a little more what you want. You'll want a `random` property that places the numbers 0 to 9 randomly? Can you give me a case where that would be useful I can't see a use case right now.

Yes, show the numbers 0 to 9 in random order each time, which is useful when used to input password.
